apps/fp-frontend-app/src/behandling/foreldrepenger/faktaPaneler/BesteberegningFaktaInitPanel.tsx

```diff
@@ -29,13 +29,17 @@ export const BesteberegningFaktaInitPanel = ({ arbeidsgiverOpplysningerPerId }:
 
   const api = useBehandlingApi(behandling);
   const { data: beregningsgrunnlag } = useQuery(api.beregningsgrunnlagOptions(behandling));
+  const skalPanelVisesIMeny =
```

Comment:
Må si at jeg synes dette er en mer kronglete if-setning enn det som var før. Men den er mer "korrekt" og i tråd med TS-typen.

Ytelsetype er en diskriminator for hvilken subtype beregningsgrunnlaget er. Når TS vet at typen er FP så vet den at besteberegninggrunnlag finnes på denne subtypen. Før var typen mindre spesifikk i den forstand at feltet fantes som possibly undefined for alle varianter. Nå vet vi at den er enten required (for FP) eller never for andre